packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -78,6 +78,7 @@ type BaseLinkCommonProps = {
    * The title of the link which is displayed as a tooltip. This is a web only prop and has no effect on react-native.
    */
   htmlTitle?: string;
+  opacity?: number;
```

Comment:
Why are changing the opacity? Can't we use a gray color that is lighter instead? Our color already has an alpha color and on top of that we're changing it more. This just means that our colors aren't sufficing our needs if we need to use opacity?

Reply:
Added same comments on figma, to be discussed with RK on monday.

Reply:
Is this discussion done? Can we move away from opacity?